README.markdown

```diff
@@ -6669,6 +6673,79 @@ This feature was first introduced in the `v0.7.3` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.shared.DICT.capacity
+------------------------
+**syntax:** *capacity_bytes = ngx.shared.DICT:capacity()*
+
+**context:** *init_by_lua&#42;, set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
+
+**requires:** `resty.core.shdict` or `resty.core`
+
+Retrieving the capacity in bytes for the shm-based dictionary [ngx.shared.DICT](#ngxshareddict) declared with
```

Comment:
Just a note: all other methods in this document are not documented with a present continuous form. Most (if not all) methods actually use the third-person form:
```
Retrieves the capacity in bytes [...]
```

Reply:
Thanks for a note. I followed `Reterieving` in [ngx.shared.DICT.get](https://github.com/openresty/lua-nginx-module#ngxshareddictget).
Should I change all `Retrieving` with `Retrieves` in all of ngx.shared.DICT methods?

Reply:
Oh, hmm. That is unfortunate (better being consistent if possible imho), but no, I don't think we should be updating any other method's description here, it isn't the scope of this PR. If present continuous form is judged acceptable, then please ignore my comment!

Reply:
Alright, I've changed `Retrieving` to `Retrieves` in doc of `capacity` and `free_space` in `ngx.shared.DICT`.